src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthRpcModule.cs

```diff
@@ -346,6 +348,7 @@ public async Task<ResultWrapper<Keccak>> eth_sendRawTransaction(byte[] transacti
         try
         {
             Transaction tx = Rlp.Decode<Transaction>(transaction, RlpBehaviors.AllowUnsigned | RlpBehaviors.SkipTypedWrapping);
+            tx.SenderAddress ??= _ecdsa.RecoverAddress(tx, true);
```

Comment:
Could you elaborate why we have to add this? 
_ecdsa.RecoverAddress(tx, true); is very expensive operation

Reply:
```
>> (4599b9fb) {"jsonrpc":"2.0","id":7,"method":"eth_sendRawTransaction","params":["0xf865808506fc23ac00830124f8940000000000000000000000000000000000000316018032a044b25a8b9b247d01586b3d59c71728ff49c9b84928d9e7fa3377ead3b5570b5da03ceac696601ff7ee6f5fe8864e2998db9babdf5eeba1a0cd5b4d44b3fcbd181b"]}
<< (4599b9fb) {"jsonrpc":"2.0","error":{"code":-32010,"message":"Value cannot be null. (Parameter 'SenderAddress')"},"id":7}
```

Reply:
why were we passing the hive tests before?